src/api/app/models/architecture.rb

```diff
@@ -16,6 +16,7 @@ class Architecture < ApplicationRecord
 
   #### Scopes (first the default_scope macro if is used)
   scope :available, -> { where(available: 1) }
+  scope :unavailable, -> { where.not(id: available) }
```

Comment:
is this a subquery? Why not plainly `where(available: 0)` ?

Reply:
Yes it was a sub-query. Changed.